src/components/Lesson.vue

```diff
@@ -265,6 +265,13 @@ export default {
   beforeMount: function () {
     this.choice = localStorage[this.cacheKey] || ''
   },
+  mounted: function () {
+    if (this.isResources) {
+      localStorage[this.lessonKey] = 'passed'
+      this.trackEvent(EVENTS.LESSON_PASSED)
+      this.updateTutorialState()
```

Comment:
I believe this line isn't necessary because `updateTutorialState` only checks real numbered lessons, not the resources lesson, so completion of the resources page has no affect on tutorial completion, right?

Reply:
correct 👌